maintenance/maintenance.sh

```diff
@@ -58,18 +58,6 @@ then
   exit
 fi
 
-
```

Comment:
This check doesn't work for our use case - ie. a service account with access to only the tools project but with rbac permissions that allow interaction with resources in other projects (ie. dev).